package.json

```diff
@@ -11,7 +11,7 @@
     "client"
   ],
   "engines": {
-    "node": ">= 10.13.0"
+    "node": ">= 12.x.x"
```

Comment:
Please use LTS here https://nodejs.org/en/blog/release/v12.13.0/

Reply:
done

Reply:
I think we should add `npm` here too to avoid oscillating `package-lock.json` between v1 and v2

Reply:
yep lets add it, huge lock diffs otherwise